doc/research/paf24/planning/ACC.md

```diff
@@ -0,0 +1,101 @@
+# ACC
+
+This file gives a general overview on possible ACC implementations and reflects the current state of the ACC.
+
+## General
+
+The main goal of an ACC (Adaptive Cruise Control) is to follow a car driving in front while keeping a safe distance to it. This can be achieved by adjusting the speed to the speed of the car in front.
+In general, classic ACC systems are designed for higher velocity (e.g. > 40 km/h). Apart from that, there are Stop & Go systems that support lower velocites (e.g. < 40 km/h). In our case, both systems might be needed and it might be reasonable to develop two different systems for ACC and Stop & Go.
+There are basically three different techniques that can be used to implement an ACC: PID Control, Model Predictive Control and Fuzzy Logic Control. Another option is CACC (Cooperative Adaptive Cruise Control) but this is not relevant for our project since it requires communication between the vehicles.
+
+### PID Control
+
+The PID Controller consists of three terms: the proportional term, the integral term and the derivative term. One possible simple controller model looks as follows:
+
+$$ v_f(t) = v_f(t - t_s) + k_p e (t-t_s) + k_d \dot{e}(t - t_s) $$
+$$ e(t-t_s) = \Delta x(t - t_s) - t_{hw,d} v_f (t - t_s) $$
+
+- $t_s$: sampling time
+- $k_p$ and $k_d$: coefficients for proportional and derivative terms
+- $e$: distance error (difference between actual distance $\Delta x$ and desired distance $\Delta x_d$)
+
```

Comment:
See Coderabbit remark. 
It is unclear which symbol means what. 
You introduced with PID but presented a PD-Formula

Reply:
Good point, I edited the formula and additional information concerning the symbols.